includes/class-response.php

```diff
@@ -231,6 +232,27 @@ public function get_header_links() {
 		return $items;
 	}
 
+	/**
+	 * Parses HTML links
+	 *
+	 * @return array
+	 */
+	public function get_html_links() {
+		$dom = $this->get_dom_document();
+		$xpath = new DOMXPath( $dom );
+		$items = array();
+
+		// check <link> and <a> elements
+		foreach ( $xpath->query( '(//link|//a)[@rel and @href]' ) as $link ) {
+			$item = array();
+			$item['rel']   = $link->getAttribute( 'rel' );
```

Comment:
Intentions

Reply:
What do you mean by intentions? I'm mirroring the header link extraction function? Technically, it extracts all rels and then matches them against the passed parameters?

Reply:
Oops, typo! Indents! 🫣

Reply:
Oh...I thought you were questioning why I did it. I really always forget to recheck after I run PHPCS Auto-fix. It sometimes messes up my indents as much as it fixes them.

Reply:
But those indents look right to me, so maybe I'm still indent-challenged.

Reply:
The equal sign? Maybe a glitch on my phone...

Reply:
Better?